README.md

```diff
@@ -80,6 +80,10 @@ Troubleshooting
 ---------------
 
 *   ```ImportError: failed to find libmagic.  Check your installation```
+    or problems unpacking a `.zip` file if you have `libmagic` version 5.39
+    (released 2020-06-16). That version does not identify ZIP file v2.0
+    correctly. This [bug][1] in `libmagic` has been reported.
+    Install the previous version 5.38 if you can.
```

Comment:
Almost perfect. :ok_hand: I would move that to a separated bullet point because this one was more specific to the absence of libmagic and provides instructions on how to install it.

After that there are "LZMA does not work", "Zstd does not work"... so you could add a point "ZIP files fail to extract" (for example) and move your detailed text there.

Reply:
Thanks, updated.

And the 5.38 pin did not work for either platform. Going to remove that for now.